phys/module_ra_gsfcsw.F

```diff
@@ -25,6 +25,7 @@ SUBROUTINE GSFCSWRAD(rthraten,gsw                              & ! PAJ: xlat and
                    ,aer_ra_feedback                               &
                    ,f_qv,f_qc,f_qr,f_qi,f_qs,f_qg,f_qndrop        &
                    ,coszen                                        & ! PAJ
+                   ,obscur                                        & ! amontornes-bcodina 2015/09 solar eclipses                                       
```

Comment:
@irowebbn @dudhia @weiwangncar 
Isaac,
This is not an optional variable. You have not modified the Registry for NMM. This would probably break the NMM build.

Reply:
Fixed.